### PR TITLE
Remove old infini modules to prepare for collection implementation

### PR DESCRIPTION
**Pull request by ekaulberg**
##### SUMMARY
Remove old infini modules which will be replaced by infinidat collection, add deprecation/redirects as appropriate

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
infini_export, infini_export_client, infini_fs, infini_host, infini_pool, infini_vol

##### ADDITIONAL INFORMATION
Following the directions from @thedoubl3j to prepare for infinidat collection implementation. This hopefully takes care of everything requested in PR1 described in [this FAQ](https://github.com/ansible-collections/overview/blob/master/README.rst#q-what-should-i-do-to-move-plugins-across-collections-during-migration).

### CODE REVIEW

**Comment by gundalow** [comment]
@ekaulberg Thank you for this PR. Can you please provide a link to the new collection repo

File: changelogs/fragments/infinidat-removal.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+removed_features:
+  - "infini_export - has been moved to the ``infinidat.infinibox`` collection. A redirection is active."
```
Comment (felixfontein) [inline changelogs/fragments/infinidat-removal.yml:2]:
```suggestion
  - "infini_* - all infinidat modules have been moved to the ``infinidat.infinibox`` collection. A redirection is active, which will be removed in version 2.0.0 (https://github.com/ansible-collections/community.general/pull/607)."
```
Please remove the other lines.

**Comment by ekaulberg** [comment]
@gundalow our collection repo is https://github.com/Infinidat/ansible-infinidat-collection

File: changelogs/fragments/infinidat-removal.yml (hunk 1 above)
Comment (ekaulberg) [reply changelogs/fragments/infinidat-removal.yml]: Done.

File: meta/runtime.yml [hunk 2]
```diff
@@ -472,3 +472,46 @@ plugin_routing:
       tombstone:
         removal_version: 1.0.0
         warning_text: The logicmonitor_facts module is no longer maintained and the API used has been disabled in 2017.
+    infini_export:
+      redirect: infinidat.infinibox.infini_export
+      tombstone:
```
Comment (felixfontein) [inline meta/runtime.yml:477]: Same for all other entries:
```suggestion
      deprecated:
```
(Sorry that I didn't notice this earlier.)

**Comment by ekaulberg** [comment]
Squashed everything into a single commit with all requested changes and no `@`s. Thank you for your patience and please advise if any further changes are needed to get this merged.

**Comment by felixfontein** [comment]
For this to get merged, the infinidat.infinibox collection needs to get released. (And for that, I guess Infinidat/ansible-infinidat-collection#1 needs to get resolved.)

**Comment by thedoubl3j** [comment]
@felixfontein ping on this, lgtm but looks like ansibull is worried about something?

**Comment by felixfontein** [comment]
@thedoubl3j looks like I got confused, it should be `deprecation`, not `deprecated`... Sorry!

**Comment by felixfontein** [comment]
(Also, there's a conflict, so this PR needs rebasing. But since it looks like it won't make it into 2.10 anyway, this can also be fixed later...)

**Pull request merged**